tsconfig.base.json

```diff
@@ -1,17 +1,16 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "esnext",
-    "lib": ["esnext"],
+    "target": "ESNext",
+    "module": "preserve",
+    "lib": ["ESNext"],
     "declaration": true,
-    "declarationMap": true,
     "emitDeclarationOnly": true,
     "declarationDir": "./dts",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
     "esModuleInterop": true,
     "isolatedModules": true,
     "allowImportingTsExtensions": true,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
```
node_modules/@types/node/events.d.ts:878:14 - error TS1291: 'events' resolves to a type and must be marked type-only in this file before re-exporting when 'isolatedModules' is enabled. Consider using 'import type' where 'events' is imported.

878     export = events;
                 ~~~~~~

node_modules/@types/node/module.d.ts:314:14 - error TS1291: 'module' resolves to a type and must be marked type-only in this file before re-exporting when 'isolatedModules' is enabled. Consider using 'import type' where 'module' is imported.

314     export = module;
                 ~~~~~~

node_modules/@types/node/stream.d.ts:1700:14 - error TS1291: 'stream' resolves to a type and must be marked type-only in this file before re-exporting when 'isolatedModules' is enabled. Consider using 'import type' where 'stream' is imported.

1700     export = stream;
                  ~~~~~~


Found 3 errors.
```

Reply:
`@types/node` is so popular that it not working is worrisome 😅

Reply:
It seems sufficient to mark the import assignment in `@types/node` as type only:
```diff
- import events = require("events");
+ import type events = require("events");
```

Can we patch this library so we don't have to turn off the lib check?

Reply:
> to mark the import assignment in `@types/node` as type only

But it is not type only. It looks like bug in TS

Reply:
Fixed in TS: https://github.com/microsoft/TypeScript/pull/57148